config/initializers/yelp.rb

```diff
@@ -0,0 +1,11 @@
+YELP_CONSUMER_KEY = "ac7g1t3S2ra7hSKSzUnBJA"
+YELP_CONSUMER_SECRET_KEY = "G5SXOND_2_fUW34keN447g5TofA"
+YELP_TOKEN_KEY = "0R49YaxmEebNFcNdEimzhyRLg463crzi"
+YELP_TOKEN_SECRET = "t84iDFoWTJXcXmT2JQg2naj4x_Y"
```

Comment:
whoops! These should be in the env file so that someone can't find these codes on github and gain control of your yelp account!